graphql/src/store/query.rs

```diff
@@ -265,6 +407,13 @@ fn build_filter_from_object(
             use self::sast::FilterOp::*;
             let (field_name, op) = sast::parse_field_as_filter(key);
 
+            let actual_entity = match is_connection_type(entity.name()) {
+                false => entity,
+                true => schema
+                    .object_or_interface(&entity.name().replace("Connection", ""))
+                    .unwrap(),
```

Comment:
`unwrap` causes a panic. It would be better to return a `QueryExecutionError` here; for other things we have `StoreError::ConstraintViolation` to report internal errors. It's probably time to add a variant `QueryExecutionError::ConstraintViolation` to report such internal errors - if we have a `None` here, clearly something is wrong in the `graph-node` code.